### PR TITLE
apptestai-test 0.0.1

### PR DESCRIPTION
**Pull request by forcemax**
TagCheck

### New Pull Request Checklist

- [x] __I will not move an already shared step version's tag to another commit__
- [x] I read the [Step Development Guideline](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md)
- [x] I have a test for my Step, which can be run with `bitrise run test` (in the step's repository)
- [x] I did run `bitrise run audit-this-step` (in the step's repository - note, if you don't have this workflow in your `bitrise.yml`, [you can copy it from the step template](https://github.com/bitrise-steplib/step-template/blob/master/bitrise.yml).)
- [x] I read and accept the [Abandoned Step policy](https://github.com/bitrise-io/bitrise-steplib#abandoned-step-policy)

### CODE REVIEW

File: steps/apptestai-test/assets/icon.svg [hunk 1]
```diff
@@ -0,0 +1,290 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" width="256px" height="256px" viewBox="0 0 256 256" enable-background="new 0 0 256 256" xml:space="preserve">  <image id="image0" width="256" height="256" x="0" y="0"
```
Comment (koral--) [inline steps/apptestai-test/assets/icon.svg]: This should be vector image, not an embedded PNG.

File: steps/apptestai-test/0.0.1/step.yml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+toolkit:
+  go:
+    package_name: github.com/forcemax/bitrise-step-apptestai-test
+deps: {}
+is_requires_admin_user: false
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- access_key: $APPTEST_AI_ACCESS_KEY
```
Comment (koral--) [inline steps/apptestai-test/0.0.1/step.yml]: Default values must not be set to custom environment variables. Only that exported by other steps, bitrise.io or Bitrise CLI are allowed.

File: steps/apptestai-test/0.0.1/step.yml [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    is_required: true
+    is_sensitive: true
+    title: apptest.ai access key (userid:accesskey). must set APPTEST_AI_ACCESS_KEY
+      env to Secrets on Workflow Editor.
+- opts:
+    is_expand: true
+    is_required: true
+    title: apptest.ai project id
+  project_id: 12389
+- binary_path: app-release.apk
```
Comment (koral--) [inline steps/apptestai-test/0.0.1/step.yml]: Similarly to comment in L35. However, here APK path exported by Gradle steps or similar can be used.

File: steps/apptestai-test/0.0.1/step.yml [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+run_if: ""
+inputs:
+- access_key: $APPTEST_AI_ACCESS_KEY
+  opts:
+    is_expand: true
+    is_required: true
+    is_sensitive: true
+    title: apptest.ai access key (userid:accesskey). must set APPTEST_AI_ACCESS_KEY
+      env to Secrets on Workflow Editor.
+- opts:
```
Comment (koral--) [inline steps/apptestai-test/0.0.1/step.yml]: It looks weird that order of elements are inconsistent across the single file. Here `opts` is first unlike 2 other values.

**Comment by forcemax** [comment]
@koral-- I will request a new pull request by correcting the points indicated in the review. Thanks.

**Pull request closed without merging**